analytics/location_heatmaps/run_experiment.py

```diff
@@ -227,6 +225,7 @@ def run_experiment(true_image,
       noiser = noise_class(dp_round_size, sensitivity, eps)
       if ignore_start_eps and start_with_level <= i:
         print_output('Ignoring eps spent', flag=output_flag)
```

Comment:
Nit: this is a frightening message; it would be nice to have a bit of extra context here (e.g., "Ignoring epsilon spent expanding first {start_with_level} levels, including current level {i}.").

Reply:
done